core/src/main/java/hudson/PluginWrapper.java

```diff
@@ -495,6 +495,17 @@ private String getVersionOf(Manifest manifest) {
         return null;
     }
 
+    /**
+     * Returns the minimumJava version of this plugin.
+     * @return the minimum Java version of this plugin.
+     *
+     * @since TODO
```

Comment:
BTW the practice has been to use `FIXME` in these comments intended to be cleaned up immediately after the merge if not in the merge commit itself, as opposed to “rainy-day” comments that could sit there for years. I guess to make it easier to grep for them.

Reply:
It _seems_ like there are actually `@since TODO` only currently in the codebase.

Anyway, I filed #3837 to address these more easily in the future.